cpp/src/bindings.cpp

```diff
@@ -340,12 +343,18 @@ PYBIND11_MODULE(_pypowsybl, m) {
                                        { sizeof(double) * m.column_count, sizeof(double) });
             });
 
-    m.def("get_sensitivity_matrix", &pypowsybl::getSensitivityMatrix, "Get sensitivity analysis result matrix for a given contingency",
+    m.def("get_sensitivity_matrix_flows", &pypowsybl::getSensitivityMatrixFlows, "Get sensitivity analysis result matrix for a given contingency",
```

Comment:
@geofjamg : do you think we should ensure retro-compatibility by keeping this kind of methods? Or is it OK in this early stage to remove some methods ?